server.js

```diff
@@ -74,10 +74,12 @@ function initialiseGlobalMiddleware (app) {
     next()
   })
 
-  app.use(function (req, res, next) {
-    noCache(res)
-    next()
-  })
+  if (process.env.NGINX_CACHING_ENABLED !== 'true') {
```

Comment:
I don't understand the meaning behind the `NGINX_CACHING_ENABLED` flag as it stands, I believe there are now 3 uses of it, one ordering this middleware and the other setting a caching cookie on the design system assets. 

Do we think this logic needs to fork on an environment flag? I believe the logic is to move the existing no cache headers to after the static assets but before the page level response is sent to allow caching while preserving the reason the no cache was initially implemented. I think this should be fine to do everywhere/ in all environments. 

It would be useful if the commit message could explain why the middleware is moved and what it means as well as what's happening.

Reply:
Just taken another quick look at this - I'm inclined to agree with Steve as it feels like moving the closure down is a bug fix of sorts, rather than trying to add or change a feature. The "no cache" closure feels like it was intended to be a "catch-all" for any dynamic content that doesn't match any of the other rules above it.